blocks/donor-wall/edit/inspector.js

```diff
@@ -27,90 +27,90 @@ const Inspector = ( { attributes, setAttributes } ) => {
 			<PanelBody title={ __( 'Donor Wall Settings' ) }>
```

Comment:
@alaca one more textdomain - I missed it at first myself.

Reply:
@kjohnson added 5846ea36